OxfordBodleian/Juel-Jensen/BDLaethf29.xml

```diff
@@ -165,62 +513,345 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                </item>
+
+                                <item xml:id="e1">
+                                    <desc>Aids for identifying weekday readings are written in the
+                                        upper margin on the incipit page of a each section of <ref
+                                            target="#ms_i1"/> in red ink
+                                        <!--framed by ornamental bands @Jacopo--> (the scribe used a
+                                        differnt nib for the reading references than for the main
+                                        text): <q xml:lang="gez">ዘሰኑይ</q> (<locus target="#15v"/>)
+                                            <q xml:lang="gez">ዘሠሉስ</q> (<locus target="#27r"/>), <q
+                                            xml:lang="gez">ዘረቡዕ</q> (<locus target="#36r"/>), <q
+                                            xml:lang="gez">ዘሐሙስ</q> (<locus target="#45v"/>), <q
+                                            xml:lang="gez">ዘዓርብ</q> (<locus target="#55v"/>), <q
+                                            xml:lang="gez">ዘቀዳሚት</q> (<locus target="#66v"/>), <q
+                                            xml:lang="gez">ዘዘወትር</q> (<locus target="#72r"/>, in
+                                        black), <q xml:lang="gez">ዘእኁድ</q> (<locus target="#74r"/>)
+                                    </desc>
+                                </item>
+
+                                <item xml:id="e2">
+                                    <desc> Scribal corrections occur occasionally over erasure (e.g.
+                                            <locus target="#112r"/> here comprising several lines)
+                                        or interlineally (e.g. <locus target="#28v #31v"/>), also
+                                        made in a secondary hand (e.g. <locus
+                                            target="#29v #40v #88v #114r"/>). Spaces (after erasure)
+                                        were filled with an interlaced band on <locus
+                                            target="#50r #72r"/> and with several thin lines on
+                                            <locus target="#66v #79v"/>. The scribe wrote the last
+                                        two lines of <ref target="#ms_i1.3"/> in small script in
+                                        order to complete the section on the same folio <locus
+                                            target="#35v"/>. Writing exercises were executed on
+                                            <locus target="#123r"/> and upside down on <locus
+                                            target="#121v #122v"/>. The name <persName
+                                            ref="PRS13017GabraMar"/>is written on <locus
+                                            target="#3r"/> (the rest of the entire folium is
```

Comment:
folio or folium?

Reply:
I would go consistently for folio? Especially since we are using "sacred/holy names" rather than _nomina sacra_?